lib/annotate/annotate_routes.rb

```diff
@@ -35,41 +35,6 @@ def content(line, maxs, options = {})
       end.join(' | ')
     end
 
-    def header(options = {})
-      routes_map = app_routes_map(options)
-
-      magic_comments_map, routes_map = extract_magic_comments_from_array(routes_map)
-
-      out = []
-
-      magic_comments_map.each do |magic_comment|
-        out << magic_comment
-      end
-      out << '' if magic_comments_map.any?
-
-      out += ["# #{options[:wrapper_open]}"] if options[:wrapper_open]
```

Comment:
I know you didn't make this change, but out of curiosity do you know if there's a reason behind using `<<` versus `+= []` for appending to an array?

Reply:
There's a semantic difference between them:
> https://stackoverflow.com/questions/4684446/why-is-the-shovel-operator-preferred-over-plus-equals-when-building-a

Reply:
That makes sense but this is mutating an array I believe. Line 43 is `out = []`

Reply:
Just tested it in Pry looks like the same applies regardless of if it's a String or an Array. Functionally it looks like one can be substituted over the other so would it makes sense to change the cases so they're the same?